server/service/devices.go

```diff
@@ -229,8 +229,8 @@ func (r *deviceAPIFeaturesRequest) deviceAuthToken() string {
 }
 
 type deviceAPIFeaturesResponse struct {
-	Err      error `json:"error,omitempty"`
-	Features fleet.DeviceAPIFeatures
+	Err      error                   `json:"error,omitempty"`
+	Features fleet.DeviceAPIFeatures `json:"features"`
```

Comment:
Just to double-check, @lucasmrod , that won't break anything on the Fleet Desktop side, right? I wanted to use the lowercase naming for consistency, I doubt it will have any ill effect given that it returns nothing at the moment (`DeviceAPIFeatures` is an empty struct), but just in case.

Reply:
> This is the one API we cannot change :)

Haha, indeed! But yeah, my thought is that it is not used as it does nothing for now, so it's the right time to fix that inconsistency.

Reply:
(that is, a repo-wide search of `/api_features` returns only the handler definition and the new docs, along with an integration test, so unless the path is built dynamically for some reason it looks safe).

Reply:
agreed, completely safe to use for now! sorry for missing that